packages/pwa/pwa-kit.config.json

```diff
@@ -1,5 +1,63 @@
 {
-    "url": {
-        "locale": "path"
+    "app": {
+        "defaultSiteId": "RefArch",
+        "url": {
+            "locale": "path",
+            "site": "path"
+        },
+        "sites": [
```

Comment:
Was it considered to retrieve this site information from SFCC via OCAPI `[/sites](https://documentation.b2c.commercecloud.salesforce.com/DOC1/topic/com.demandware.dochelp/OCAPI/current/shop/Resources/Site.html#id1675432766__id-992161710)` call?

Reply:
Hey @stm-dschulz, thank you for bringing that up, currently, we are not reaching out to any endpoint for the site data, it is manually input. However, it is the next thing we will look at after we finalize our implementation.